src/modes/map/map_treasure.cpp

```diff
@@ -319,14 +319,14 @@ void TreasureSupervisor::_UpdateList()
             _selection_name.SetText(UTranslate("Drunes"));
             _selection_icon = GlobalManager->Media().GetDrunesIcon();
 
-            // Plural gettext for the 1st sentence.
             const std::string temp_string =
+                // Part 2 of "With the additional %u drune(s) found in this treasure added, ... "
```

Comment:
fair enough, I didn't know you were speaking for translator comments as we didn't agree on the tr: format yet. I'll merge for now.